central/scannerdefinitions/handler/handler.go

```diff
@@ -65,6 +68,8 @@ var (
 		"name2repos": "repomapping-tmp/container-name-repos-map.json",
 		"repo2cpe":   "repomapping-tmp/repository-to-cpe.json",
 	}
+
+	v4VersionRegex = regexp.MustCompile(`^(?:\d+\.\d+\.\d+(?:-rc\.\d+)?(?:-\d+-g[0-9a-f]+|-nightly-\d{8})?|dev)$`)
```

Comment:
if you want, you can make `getV4Files` take in some custom enum type instead of `updaterKey string` to help differentiate how to handle each type of query